modules/cactus-web/src/Header/Header.tsx

```diff
@@ -0,0 +1,112 @@
+import PropTypes from 'prop-types'
+import React, { Children, ReactNode } from 'react'
+import styled from 'styled-components'
+import { margin, MarginProps } from 'styled-system'
+
+import Text from '../Text/Text'
+
+export type BackgroundColorVariants = 'lightContrast' | 'white'
+interface HeaderProps extends MarginProps, React.HTMLAttributes<HTMLDivElement> {
+  bgColor?: BackgroundColorVariants
+  children?: ReactNode
+}
+
+export type HeaderType = React.FC<HeaderProps> & {
+  Item: typeof HeaderItem
+  BreadcrumbRow: typeof HeaderBreadcrumbRow
+  Title: typeof HeaderTitle
+}
+
+export const HeaderItem: React.FC<React.HTMLAttributes<HTMLDivElement>> = ({ children }) => {
```

Comment:
The type signature here is not valid.  Since this element is not a div, it shouldn't accept div props.  If you want it to render a div, you could change it to simply: `export const HeaderItem: React.FC<React.HTMLAttributes<HTMLDivElement>> = (props) => <div {...props} />`.  If not, then I'd change this to:

```suggestion
export const HeaderItem = ({ children }: { children?: ReactNode }): React.ReactElement => <>{children}</>
```

Reply:
@mikesoltow That's a lot better than what I've done here. There is a problem here though when I implement your suggestion: neither the BreadcrumbRow and the Items are being rendered. 

That's weird, I will take a look to find why is this happening.

Reply:
I think it might be a weird TypeScript thing.  Try changing it to:

```
export const HeaderItem: FC = ({ children }) => <>{children}</>
```